### PR TITLE
feat(dupconst-cop): customcop to check dup values for constants

### PR DESCRIPTION
**Pull request by PrajnaYaji18**
https://github.com/amagimedia/blip/pull/7440

### CODE REVIEW

File: lib/cops/readme.md [hunk 1]
```diff
@@ -19,3 +19,39 @@ Enabling the cop inside .rubocop.yml :
...
+~~~ruby
+ # allowed
+ TEST = 10
+ TEST1 = 10
+~~~
+
+~~~ruby
+ # allowed
+ TEST = ["abc", "def"]
+ TEST1 = "abc", "def"]
```
Comment (SowmyaAmagi) [inline lib/cops/readme.md]: typo?

**Review by SowmyaAmagi — COMMENTED** [review]
LGTM

File: lib/cops/readme.md [hunk 2]
```diff
@@ -19,3 +19,39 @@ Enabling the cop inside .rubocop.yml :
 CustomCop/PutLoggerFormatCop:
   Enabled: true
 ~~~
+
+### DuplicateConstantCop
+DuplicateConstantCop ensures that there is no two constants in a file which has same string value
```
Comment (rparjun) [inline lib/cops/readme.md:24]: can you add an example of why it is not recommented to have the same value for constants?

**Comment by rparjun** [comment]
LGTM

**Comment by rparjun** [comment]
Please merge this and tag v1.0.0

**Pull request merged**